argo/kfp-compiler/resources/compile.sh

```diff
@@ -2,6 +2,6 @@
 
 PYTHON_VERSION=$(python3 -V | grep -Eo '[0-9]+\.[0-9]+')
 
-export PYTHONPATH="$PYTHONPATH:/$(dirname $0)/py$PYTHON_VERSION"
+export PYTHONPATH="$PYTHONPATH$(dirname $0)/py$PYTHON_VERSION"
```

Comment:
When concatenating paths, we should always separate with an additional `:` character 
```suggestion
export PYTHONPATH="$PYTHONPATH:$(dirname $0)/py$PYTHON_VERSION"
```

Reply:
noticed `:://` in actual PYTHONPATH when running - it obviously did 'work' before, but corrected it anyway. maybe not the best fix.

Reply:
sure, but that was because the original `PYTHONPATH` ended in a `:` already - which is not the case usually. Most likely, an empty string got concatenated before. Best practice for paths is always to concatenate with a new `:` so that we don't have to make assumptions about how it was set before.

Reply:
ok

Reply:
done